docs/supported_apis/index.rst

```diff
@@ -33,20 +33,17 @@ Modin.
 
 The exact methods we have implemented are listed in the respective subsections:
 
-* DataFrame_
-* Series_
-* utilities_
-* `I/O`_
+* :doc:`DataFrame </supported_apis/dataframe_supported>`
+* :doc:`Series </supported_apis/series_supported>`
+* :doc:`utilities </supported_apis/utilities_supported>`
+* :doc:`I/O </supported_apis/io_supported>`
 
 We have taken a community-driven approach to implementing new methods. We did a `study
 on pandas usage`_ to learn what the most-used APIs are. Modin currently supports **93%**
 of the pandas API based on our study of pandas usage, and we are actively expanding the
 API.
 
-.. _DataFrame: dataframe_supported.html
-.. _Series: series_supported.html
-.. _utilities: utilities_supported.html
-.. _I/O: io_supported.html
-.. _study on pandas usage: https://github.com/modin-project/study_kaggle_usage
```

Comment:
Why is this removed?

Reply:
`DataFrame`, `Series`, `utilities` and `I/O` was changed using `:doc:` format. `study on pandas usage` is saved.